app/javascript/components/workflows/student/exams/show/components/FileViewer.tsx

```diff
@@ -207,15 +207,15 @@ export const ControlledFileViewer: React.FC<ControlledFileViewerProps> = (props)
         <Col sm={3} className="overflow-scroll-x">
           <FileTree
             files={filteredFiles}
-            selectedFile={selection?.selectedFile ?? firstID}
+            selectedFile={selection?.selectedFile ?? null}
```

Comment:
This seems redundant -- does react treat null differently from undefined?

Reply:
`undefined` is the same as not passing the prop.
The Material Design treeview will then think it is uncontrolled, and complain about moving from controlled to uncontrolled when the first file is selected.